baseplate/lib/log_formatter.py

```diff
@@ -0,0 +1,9 @@
+from pythonjsonlogger import jsonlogger
+
+
+class CustomJsonFormatter(jsonlogger.JsonFormatter):
+    def process_log_record(self, log_record: dict) -> dict:
+        log_record["level"] = log_record.pop("levelname", None)
+        if "traceID" not in log_record and "threadName" in log_record:
+            log_record["traceID"] = log_record.get("threadName", None)
```

Comment:
you need to make sure it's an integer to call it a traceID. there can be greenlets with other thread names that would make traceID inaccurate.